src/Domain/Supplier.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Linn.Tax.Domain
+{
+    public class Supplier
+    {
+        public int SupplierId { get; set; }
+
+        public string LiveOnOracle { get; set; }
```

Comment:
I don't know if you're using this class at the moment but you don't need LiveOnOracle. It's still in various queries because it was needed when switching from vax to oracle but you can take it out of anything written now.